Test/DependencyCollector/FunctionalTests/FuncTest/SqlTests.cs

```diff
@@ -23,6 +24,11 @@ public class SqlTests
         private const string ResourceNameSQLToDevApm = @".\SQLEXPRESS | RDDTestDatabase";
 
         /// <summary>
+        /// Resource Name for wrong database.
+        /// </summary>
+        private const string ResourceNameSQLToWrongDatabase = @"invalid\SQLEXPRESS | RDDTestDatabase";
```

Comment:
Yes, have tested different cases manually. Added 'invalid account' case in functional tests.